cli/azd/pkg/azure/resource_ids.go

```diff
@@ -51,6 +51,15 @@ func WebsiteRID(subscriptionId, resourceGroupName, websiteName string) string {
 	return returnValue
 }
 
+func AksRID(subscriptionId, resourceGroupName, clusterName string) string {
```

Comment:
It looks like this is the same as `KubernetesServiceRID`.  Is there a reason to have both?

Reply:
lol - no.  Just me doing work at different times and forgot I already did it.  Will remove :)